adot/collector/lambdacomponents/go.mod

```diff
@@ -0,0 +1,134 @@
+module lambdacomponents
+
+go 1.18
+
+retract v0.25.0
```

Comment:
Is this necessary anymore? This will essentially be a completely new package under `aws-otel-lambda`. `v0.25.0` does not exist here.

Reply:
I agree. Updated the PR